README.md

```diff
@@ -1,2 +1,74 @@
-# oam-runtime
-A set of libraries for building OAM runtimes
+# oam-kubernetes-runtime Project
+
+The OAM Kubernetes runtime project is a collection of Golang help libraries and official api type
+ definitions for building an OAM Kubernetes runtime that is compatible with the latest stable
+  OAM spec. 
+
+## Prerequisites
+
+1. go version 1.12+
```

Comment:
I believe we are using go `1.13` here, but I don't think this a strict prerequisite for consuming?

Reply:
I believe this is related to the go module usage.  Maybe go 1.11 also works but I haven't tried.